spec/features/edit_dataset_spec.rb

```diff
@@ -19,8 +19,6 @@
                                                  owner: user ) }
 
   before(:each) do
-    allow_any_instance_of(UrlValidator).to receive(:validPath?).and_return(true)
```

Comment:
This seems unrelated to your commit?

Reply:
It's duplicated from the spec_helper.

Reply:
I don't think anyone's going to miss it ;-)